### DISCUSSION TITLE
a button to save article and keep editing

### DISCUSSION THREAD

**Discussion by ShulyAvraham** [forem/forem#19068] [Feature Requests]
Following the discussion for auto save #15862 - I propose a simpler solution (maybe) to have a button to save the article while staying in the edit mode, and keep the editor open.

**Comment by jamesbro123** [comment]
Hi,
In response to the discussion regarding auto save feature #15862, I propose a simpler solution that may address the issue. Instead of relying on auto save, we can implement a button that allows users to manually save their articles while staying in edit mode. This way, users can have more control over the saving process and ensure that their changes are being saved properly.

Furthermore, this proposed solution would allow users to keep the editor open after saving their article, which can be helpful when making additional changes or revisions. Overall, implementing a button to save game articles site or any [mod blog](https://modrar.com/) while keeping the editor open can provide a more streamlined and efficient editing experience for users.
